js/renovation/ui/form/form_props.ts

```diff
@@ -1,6 +1,9 @@
-import { ComponentBindings } from '@devextreme-generator/declarations';
+import { ComponentBindings, OneWay } from '@devextreme-generator/declarations';
 
 @ComponentBindings()
 export class FormProps {
+  @OneWay() scrollingEnabled = false;
 
+  // TODO: seems is not used
+  @OneWay() useNativeScrolling?: boolean;
```

Comment:
Let's add it later with a code that uses it

Reply:
It uses for preventing simulated scroll bars visibility for android devices